### DISCUSSION TITLE
Inquire about  budibase trial from Dairy Plus company Thailand

### DISCUSSION THREAD

**Discussion by LTCTHITA** [Budibase/budibase#11342] [Help] [answered]
- As Mr. Mike used to send a Examples of programs related to name search in hospitals to try to write the program accordingly. When trying to write accordingly but the written button has no action So I want to know why and how to fix it.
 
![Capture 1](https://github.com/Budibase/budibase/assets/93240913/83bc22f9-d21b-44c3-9ee4-6b00941cd8ed)
![image](https://github.com/Budibase/budibase/assets/93240913/3e9a253c-9ef7-4e96-bd98-0cc40f40007e)
 
- My Usage says Monthly limits.
RESETS IN 7 DAYS means I will need to upgrade, right?
Can you help me to estimate the price and make a quotation? For about 200 User usage, users will be divided into groups.

![Capture 3](https://github.com/Budibase/budibase/assets/93240913/441a539c-60c1-4a0b-b2c9-ba382102222c)

**Comment by ConorWebb96** [comment, marked as answer]
Hey @LTCTHITA,

I'd recommend using the form here on the Budibase website; https://budibase.com/contact/ after filling in all the relevant information one of the sales engineers will be in touch.
